Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Android/Linker/LinkerPleaseInclude.cs

```diff
@@ -0,0 +1,85 @@
+using System.Collections.Specialized;
+using System.Windows.Input;
+using Android.App;
+using Android.Views;
+using Android.Widget;
+
+namespace MvvmSeed.Android.Linker
+{
+    /// <summary>
+    /// This file is required to avoid android Linker wiping references to code not detected during static analysis.
+    /// (We could disable linking, but I rather not, no linking means x3 or x4 size for builds)
+    /// </summary>
+    public class LinkerPleaseInclude
```

Comment:
Nope, I think you got it right.

I am also not a big fan of having a huge code behind with bindings, I definitely think XAML is the place for that.

Maybe we can find / write something that will link the right classes by looking at the XAML bindings?

What I'd really like to avoid is someone needing to remember to add a line into this class because they forgot before and the released version of the app does not work. The more automatic, the better.